Founding-membership.md

```diff
@@ -0,0 +1,67 @@
+# Become a founding member
+
+We’re currently looking for founding members to help build the Foundation for Public Code’s long term future by joining us now.
+
+Founding members are forward-thinking public organisations with a strong principled commitment to the existence of an ecosystem of public code. You’ll be willing to take co-ownership of the Foundation [in line with our draft principles] and be an advocate for public code.
+
+We are looking for organisations that can either:
+
+* help us validate our code certification and stewardship processes
+* provide material support to create a public code ecosystem
+
+We’ll open other types of membership from 2020.
+
+## Benefits of being a founding member
+
+As a founding member, you’ll shape the Foundation’s processes and governance.
+
+As a founding member, you will:
+
+* be one of the first to submit your codebases for stewardship by the Foundation (reducing your future risk and maintenance costs)
+* get a seat on the board of directors
+* have your membership fees waived for the first year
```

Comment:
I don't think this should be a standard offer, since we're going to have to ask Founding members to fund us, often the first year already since we don't have money until the end of 2020.

Reply:
I've removed this.